src/pages/_id/index.vue

```diff
@@ -230,17 +256,37 @@ export default {
     error({ statusCode: 404, message: 'LIKER_NOT_FOUND' });
     return undefined;
   },
+  created() {
+    if (
+      this.currentTab !== 'created' &&
+      this.$route.hash === this.creatorFollowSectionHash
+    ) {
+      // NOTE: `route.hash` never sent to server side,
+      // Therefore, it is done in here
+      // Ref: https://www.rfc-editor.org/rfc/rfc2396#section-4
```

Comment:
created is also a universal hook that gets called by server side, if we want client only we should use `mounted`